### PR TITLE
Milestone 1: Create Book class

### LINKED ISSUE #1: [1pt] TM #1 Create Book class
- Create a Book class in a separate .rb fileDescription to be added
- All Book class properties visible in the diagram should be defined and set up in the constructor method.

### PR DESCRIPTION
**Pull request by skabeo**
In this pull request I;

- Created a book class in `classes/book.rb` file
- It inherited the item class
- All book class properties in the diagram was implemented

### CODE REVIEW

**Comment by kwesiObuobi** [comment]
Hello, @skabeo 
It seems you forgot to pass the `published_date` and `archived` properties into the `Book` constructor

File: classes/book.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require_relative 'item'
+
+class Book < Item
+  def initialize(publisher, cover_state)
+    super(published_date, archived)
```
Comment (kwesiObuobi) [inline classes/book.rb]: @skabeo , the `published_date` and `archived` properties don't yet exist in this context.
I know you only forgot to pass them into the `Book` constructor, before sending them to `super()` 😄

**Review by kwesiObuobi — CHANGES_REQUESTED** [review]
Kindly review the `Book` constructor

File: classes/book.rb (hunk 1 above)
Comment (skabeo) [reply classes/book.rb]: Thank you. I'm on it

Comment (skabeo) [reply classes/book.rb]: Book constructor changes implemented

**Review by kwesiObuobi — APPROVED** [review]
Hi @skabeo  ,

STATUS: APPROVED! 🥇 💯
approved-accepted

Congratulations on completing this task🎉

### Highlights
- No linter errors. ✅
- followed git workflow. ✅

File: classes/book.rb (hunk 1 above)
Comment (kwesiObuobi) [reply classes/book.rb]: 👍🏽

**Review by Yacoubou-seidou — APPROVED** [review]
Hi @skabeo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights 
:+1: create books class

Cheers and Happy coding!👏👏👏_

**Pull request merged**